R4ML/R/util.io.R

```diff
@@ -212,11 +212,11 @@ is.r4ml.fs.valid <- function() {
 }
 
 #' r4ml.read.csv
-#' @description Returns a R data.frame in local mode or a SparkR DataFrame in cluster mode.
-#' @param file path to the input file
-#' @param header logical
-#' @param stringsAsFactors logical
-#' @param inferSchema logical
+#' @description Returns an R data.frame in local mode or a SparkR DataFrame in cluster mode.
```

Comment:
I don't think we should say `SparkR DataFrame`. See my above comment about using either `Spark DataFrame` or `SparkDataFrame`.